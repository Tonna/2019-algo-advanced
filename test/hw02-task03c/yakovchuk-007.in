13
13 9 1 8 11 3 2 6 10 12 4 7 5