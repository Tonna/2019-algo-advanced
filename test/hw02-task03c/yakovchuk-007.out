3 7 3 11 13 11 11 9 11 12 11 13 13
